python-frontend/src/main/java/org/sonar/python/semantic/ProjectLevelSymbolTable.java

```diff
@@ -84,10 +91,20 @@ public void addModule(FileInput fileInput, String packageName, PythonFile python
       }
     }
     globalSymbolsByModuleName.put(fullyQualifiedModuleName, globalSymbols);
+    if (globalSymbolsByFQN != null) {
+      addModuleToGlobalSymbolsByFQN(globalSymbols);
```

Comment:
I'm thinking we could probably always add symbols to `globalSymbolsByFQN` incrementally so that `globalSymbolsByFQN` would just become a getter. However for the time being we can keep it as you did. If you also agree, can you add a TODO comment?